e2e/tests/api-driven/package.json

```diff
@@ -7,8 +7,8 @@
   "packageManager": "pnpm@8.6.6",
   "dependencies": {
     "@cucumber/cucumber": "^9.3.0",
-    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#6c2cc59",
-    "axios": "^1.7.2",
+    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#fc14234",
+    "axios": "^1.6.8",
```

Comment:
This looks like a rebasing bug - we want to keep `axios` at v1.7.2 please as merged this morning because that fixed e2e issues !